### ISSUE TITLE
6.005 S16 syllabus didn't import

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-to-hugo#472]
### Steps to Reproduce

legacy: https://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-005-software-construction-spring-2016/syllabus/
nextgen: https://ocwnext.odl.mit.edu/courses/6-005-software-construction-spring-2016/pages/syllabus/
github: https://github.mit.edu/mitocwcontent/6.005-spring-2016/blob/main/content/pages/syllabus.md

**Comment by Wassaf-Shahzad** [comment]
@pdpinch 
Syllabus is rendering locally for me.
<img width="1440" alt="Screenshot 2022-03-01 at 5 35 03 PM" src="https://user-images.githubusercontent.com/87968618/156170063-46cd5d61-e34f-4c4f-9f97-2df9ed5d9c4f.png">

**Comment by pdpinch** [comment]
Can you tell me how you got that? Did you run it through ocw-to-hugo again?

**Comment by Wassaf-Shahzad** [comment]
> Can you tell me how you got that? Did you run it through ocw-to-hugo again?

Yes I just downloaded the course from aws using ocw-to-hugo

**Comment by pdpinch** [comment]
Please consult with @gumaerc or @mbertrand about how to to

- [ ] update the ocw-to-hugo output for this course
- [ ] how to import this page into studio, without overwriting the rest of the course (which may have been editing in studio already)

If there's no way to import only one page, it may be easier to just create the page through the web ui and paste in the content.

**Comment by gumaerc** [comment]
@pdpinch @Wassaf-Shahzad It seems that whatever was causing this page to not properly convert has been fixed as the markdown for it in `ocw-to-hugo-output-production` is fully populated.  Peter is correct that there isn't a management command currently to import a single page.  The best approach is indeed to just update pages like this by pasting the markdown into Django admin, save and then publish.  I did this for this course and the syllabus page is now rendering: https://ocwnext.odl.mit.edu/courses/6-005-software-construction-spring-2016/pages/syllabus/
